dialog.php

```diff
@@ -19,6 +19,9 @@
             case 'diff':
                 include('templates/diff.html');
                 break;
+            case 'fetch':
+                include('templates/fetch.html');
```

Comment:
I will Implement into the remotes dialog (and also for overview dialog for <default>.)